cart.go

```diff
@@ -255,14 +252,18 @@ func main() {
 	// Get artifact from buildNum
 	u := expansions.ExpandURL(artifactsURL)
 	verboseln("Artifact list:", censorURL(u))
-	resBody := httpClient(u, circleToken)
-
-	var items artifactItems
+	res := httpGet(u, circleToken)
 
-	if err := json.Unmarshal(resBody, &items); err != nil {
+	defer res.Body.Close()
+	body := new(bytes.Buffer)
+	if _, err := io.Copy(body, res.Body); err != nil {
```

Comment:
Since this is just unmarshaled, you can just create a `json.Decoder` and decode right from `res.Body`. No need to buffer here.